controller/app.controller.js

```diff
@@ -1,13 +1,18 @@
-const { gettingTopics, gettingDescriptions } = require("../modele/app.model");
+const { gettingTopics, gettingArticles } = require("../modele/app.model");
 const endpoints = require("../endpoints.json");
+const { articleData } = require("../db/data/test-data");
 exports.getTopics = (req, res) => {
   return gettingTopics().then((topics) => {
     return res.status(200).send({ topics: topics });
   });
 };
 exports.getDescriptions = (req, res) => {
-  console.log("controller");
-
-
   return res.status(200).send({ description: endpoints });
 };
+exports.getArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  return gettingArticles(article_id).then((article) => {
```

Comment:
I find the variable naming a little confusing getArticle and gettingArticles are quite similar. The model is also plural which implies that it is getting multiple items when it isn't. I tend to name my models after the SQL statement (eg selectArticle). What ever convention you use is fine as long as it is consistent throughout your app.